faculty/clients/job.py

```diff
@@ -57,6 +62,26 @@ class RunState(Enum):
     ["name", "description", "author_id", "created_at", "last_updated_at"],
 )
 JobSummary = namedtuple("JobSummary", ["id", "metadata"])
+InstanceSize = namedtuple("InstanceSize", ["milli_cpus", "memory_mb"])
+JobParameter = namedtuple(
+    "JobParameter", ["name", "param_type", "default", "required"]
+)
+JobCommand = namedtuple("JobCommand", ["name", "parameters"])
+JobDefinition = namedtuple(
+    "JobDefinition",
+    [
+        "working_dir",
+        "command",
+        "image_type",
+        "conda_environment",
+        "environment_ids",
+        "instance_size_type",
+        "instance_size",
+        "max_runtime_seconds",
+    ],
+)
+Job = namedtuple("Job", ["meta", "definition"])
```

Comment:
From looking at the Scala code, it looks like we also return the job ID with the job as the `jobId` key. Can we add it here as:
```suggestion
Job = namedtuple("Job", ["id", "meta", "definition"])
```
And then extract it in the schema with `data_key="jobId"`?